### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by axefrog** [alefragnani/vscode-bookmarks#161]
Came here to report that the bookmarks sidebar is ugly and could do with some improvements, but noticed almost no commits in a very long time...

![image](https://user-images.githubusercontent.com/298883/48809991-ada31180-ed72-11e8-84ea-6e2049a545f1.png)

**Comment by alefragnani** [comment]
Hi @axefrog , that was fun 😄 ! 

Maybe you didn't notice, but I released an update 3 days ago, with 3 new features. So, **Yes, It Is Alive!**

![image](https://user-images.githubusercontent.com/3781424/48813416-f7154180-ed1d-11e8-82a6-7bc10ac8a68b.png)

About being _ugly_, well, that's an opinion. Could you explain _why_ do you think this is _ugly_, and how you could make it _prettier_, besides the already reported issues, like #143, #145 and #149?

But, if you are asking the reason why _these features_ hasn't been released yet, or if you expected more _frequent releases_, well, that's another story. 

TLDR, A few weeks ago I decided to "re-focus" on some of my projects, and now its evolutions will be based on **community feedback and support**. So, to decide whether a new feature worth to be developed, I need to know if _there is enough people interested_ and/or _the project is [financially supported](https://github.com/alefragnani/vscode-bookmarks#support) by the users_.

So, if you would like to see more frequent updates, or some of these features being released, feel free to upvote (👍 ) the issues and [support the project](https://github.com/alefragnani/vscode-bookmarks#support), becoming [Become a Patron](https://www.patreon.com/alefragnani). 

Thank you

**Comment by axefrog** [comment]
Hey, sorry, not my intent to offend, I just didn't take the time to break things down as the project seemed inactive and wasn't sure anyone would respond. We're all busy, and I am certainly not one to demand anything from someone offering something for free :-)

With respect to what I *actually* meant by "ugly", I was hoping to use bookmarks to help me break a large file into discrete, labelled chunks, but didn't end up using the extension as I felt the sidebar wasn't easy enough to scan at a glance. What do I mean by this? Well, the most important information is the label, but it there is no visual break between the label and the location information, which makes the list a lot more visuall cluttered than it needs to be. Also, there are two icons on every row - is the pencil icon necessary? On Every row the second icon is a pencil icon, which suggests that perhaps it is just taking up space.

As a quick mockup, this would be my suggestion for an improved look:

![image](https://user-images.githubusercontent.com/298883/48832114-28484d00-edc4-11e8-880f-006187f69e5c.png)

In actual fact I only originally visited the repo to see if there was a roadmap or equivalent.

**Comment by alefragnani** [comment]
That's ok. I'm glad you did understand 👍  ...

About the mockup, VS Code does not support such customisation. The tree items are always "flat" with no kind of styling. In the more recent release (1.28), a new API is being proposed (
https://code.visualstudio.com/updates/v1_29#_proposed-extension-apis) but only to add a _"highlight"_ style. No **bold**, _italic_, etc. So, we are out of luck.

The pencil icon is used to represent a labeled bookmark, becoming distinguished from the default bookmarks, which shows the line contents. Maybe, instead of adding the pencil icon side by side to the bookmark icon, it would "replace it". You end up with only one row of icons.

Hope this helps.

**Comment by axefrog** [comment]
Another option could be to shift the extraneous metadata to child items in the tree (collapsed by default), or put them in a tooltip?

**Comment by alefragnani** [comment]
Do you mean `Ln: XX, Col: YY)` as metadata?  I prefer using tooltip instead of child items, and if I remember correctly, there is a API available. For this, there is already #143, so I should simply update its descriptions.

**Comment by alefragnani** [comment]
Closing this issue in favor of #143.
